src/utilities/attachments-helper.ts

```diff
@@ -9,7 +9,9 @@ interface AttachmentHelperOptions {
 };
 
 interface RequestPresignedURLResponse {
-    upload_url: string;
+    // TODO: URL type?
```

Comment:
We shouldn't be doing any backend logic with the url since it is now passing the filename separate so it doesn't add any value

We could validate that it is a valid url but what if in the future it was a relative url (like we used the same cloudfront base for our uploads

I can see either the backend or frontend validating the urls integrity (especially now that the frontend reports errors back)

Reply:
Preventative typing doesn't have to be used immediately for it to add value. But you did use extra logic with when you were parsing out the filename (though this typing wouldn't strictly help with that either).

In general, I think we prefer to err on the side of specificity with our typings. That's one of the benefits, having to explicitly change the types when you're changing the logic.